systems/robotInterfaces/@Biped/Biped.m

```diff
@@ -221,6 +221,20 @@
     function bool = isRightSupport(obj,rigid_body_support_state)
       bool = ~any(rigid_body_support_state.bodies==obj.robot.foot_body_id.left) && any(rigid_body_support_state.bodies==obj.robot.foot_body_id.right);
     end
+
+    function fc = getFootContacts(obj, q)
```

Comment:
can you add a quick doc for this?  by the name I would have thought it woudl return all of the contact points on the feet, but looking at the code I see that it returns only those in collision.